Eshop.Domain/Product.cs

```diff
@@ -2,36 +2,40 @@
 {
     public class Product
     {
-        public Product(int id, string title, string description, decimal price)
+        public Product(int id, string title, string description, decimal price, Category category)
         {
             if (id < 0)
             {
                 throw new ArgumentOutOfRangeException("id");
             }
 
-            ValidateParameters(title, description, price);
+            ValidateParameters(title, description, price, category);
 
             Id = id;
             Title = title;
             Description = description;
             Price = price;
+            Category = category;
         }
 
         public int Id { get; private set; }
```

Comment:
remove from Id private set; - check video to understand why

Reply:
fixed properly